tests/integration/eoa/test_ethaa.py

```diff
@@ -0,0 +1,134 @@
+import os
+import sys
+
+import pytest
+import web3
+from eth_account._utils.legacy_transactions import (
+    serializable_unsigned_transaction_from_dict,
+)
+from eth_keys import keys
+from starkware.starknet.testing.starknet import Starknet
+from starkware.starkware_utils.error_handling import StarkException
+
+from tests.integration.helpers.helpers import int_to_uint256
+from tests.utils.signer import MockEthSigner
+
+
+@pytest.mark.asyncio
+class TestExternallyOwnedAccount:
+    class TestComputeStarknetAddress:
+        async def test_should_return_same_address_as_contract(
+            self, externally_owned_account, addresses
+        ):
+            (
+                deployer,
+                evm_eoas,
+            ) = externally_owned_account
+
+            for i in range(0, len(evm_eoas)):
```

Comment:
when you want to make several similar tests, use `@pytest.mark.parametrize` and not an inside loop, otherwise you loose the info about what fails and what is ok (and cannot run one single case neither)